schemas/assets/annotation.schema.json

```diff
@@ -0,0 +1,360 @@
+        "adbcollab:type": {
+          "title": "Annotation Type",
+          "description": "node subtype identifies a global annotation that is made on an entire artboard, or page of a document. Such a node annotation does not need noteX, noteY properties described below.\nnote subtype is similar to a sticky note in PDF - it is a contextual annotation made at (x, y) location on an artboard or page of a document.\nstrikethrough | highlight | underline | insert subtype correspond to namesake annotation usually made against text in PDF.\nshape annotation is a free form vector drawing annotation.",
+          "type": "string",
+          "meta:enum": ["node", "note", "strikethrough", "highlight", "underline", "insert", "shape"]
+        },
+        "adbcollab:primarySelector": {
+          "title": "Primary Selector",
+          "description": "annotationSelector does not extend annotationSelectorChooser but composes via primarySelector. This helps if the actual selection requires a collection of selections, possible in adobeNodeSelector.",
+          "$ref" : "#/definitions/annotationSelectorChooser"
+        },
+        "adbcollab:refinedBy": {
+          "title": "Refined By",
+          "description": "An annotation may be defined for multiple representations or a specific context of the asset. Example 1: In case of PDF, an annotation may be present at a specific location in a page, or across two pages. A PDF could be rendered/reflowed by relayouting on a smaller form factor (mobile) device. To enable this feature, annotation may need to be represented via text positions and an anchor element/id. Example 2: In case of Adobe XD, an annotation may be present on an overlay artboard when it is superimposed on top of a base artboard. So, the annotation may need both adbcollab:primarySelector and adbcollab:refinedBy to represent it. adbcollab:refinedBy is only present if adbcollab:primarySelector is present.",
+          "$ref" : "#/definitions/annotationSelectorChooser"
+        }
+      }
+    },
+    "annotationStyle": {
+      "title": "Annotation Selector",
+      "description": "An annotation may refer to segment of a target resource, rather than all of it. A selector is used to describe that segment in the target resource.",
+      "type": "object",
+      "properties": {
+        "type" : {
+          "type": "string",
+          "const": "AdobeAnnotationStyle"
+        },
+        "adbcollab:strokeColor": {
+          "title": "Stroke color",
+          "description": "The CSS color for stroke drawing current annotation object",
+          "type": "string",
+          "examples": ["#ffd300"]
+        },
+        "adbcollab:strokeWidth": {
+          "title": "Stroke width",
+          "description": "stroke width for drawing the annotation",
+          "type": "number",
+          "default": 1
+        },
+        "adbcollab:opacity": {
+          "title": "Opacity",
+          "description": "Fill and stroke opacity of the annotation",
```

Comment:
This is similar to CA entry in Table 170 in PDF 32000-1:2008 
(Optional; PDF 1.4) The constant opacity value that shall be used in painting the annotation (see Sections 11.2, “Overview of Transparency,”and 11.3.7, “Shape and Opacity Computations”). This value shall apply to all visible elements of the annotation in its closed state.
--